conf/modules.config

```diff
@@ -85,9 +85,16 @@ process {
     withName: "BUSCO_BUSCO" {
         // Obey "use_work_dir_as_temp", except for large genomes
         scratch = { !params.use_work_dir_as_temp || (meta.genome_size < 2000000000) }
-        ext.args = { 'test' in workflow.profile.tokenize(',') || 'test_raw' in workflow.profile.tokenize(',') || 'test_nobusco' in workflow.profile.tokenize(',') ?
-                        '--force'  // Runs with miniprot
-                    : '--force --metaeuk' }
+        ext.args = {
+            def base = '--force'
+            if (params.busco_gene_predictor == 'augustus') {
+                base += ' --augustus'
+            } else if (params.busco_gene_predictor == 'metaeuk') {
+            base += ' --metaeuk'
+            }
+            // if it's miniprot, do nothing extra
```

Comment:
```suggestion
            // is a certain predictor requested ?
            if (params.busco_gene_predictor) {
                base += ' --' + params.busco_gene_predictor
            }
            // otherwise, let's go with the default (miniprot)
```